packages/steps/src/index.vue

```diff
@@ -7,6 +7,8 @@
 <script lang="ts">
 import { defineComponent, watch, ref, provide } from 'vue'
 
+const CHANGE_EVENT = 'change'
```

Comment:
I need you to move this to `utils/constants`, later we will replace all the event constants with this

Reply:
OK, I will add it later.